content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,568 @@
+  - name: Arduino_EdgeControl
+    url: https://github.com/arduino-libraries/Arduino_EdgeControl
+  - name: ArduinoJson
+    url: https://github.com/bblanchon/ArduinoJson
+  - name: Arduino_ConnectionsHandler
+    url: https://github.com/arduino-libraries/Arduino_ConnectionHandler
+  - name: ArduinoIoTCloud
+    url: https://github.com/arduino-libraries/ArduinoIoTCloud
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+  - arduino-agent
+hardware:
+  - hardware/05.pro-solutions/boards/solutions-and-kits/edge-control
+  - hardware/05.pro-solutions/boards/solutions-and-kits/enclosure-kit
+  - hardware/01.mkr/01.boards/mkr-wan-1310
+---
+
+## Introduction
+
+We know that agricultural activities are normally carried out in remote environments, this makes access to electricity and connectivity a challenge. 
+
+Smart farming techniques are being implemented more and more due to the importance of optimizing the use of resources. This includes the demand for more efficient, eco-friendly, and more profitable crops.
+
+![Application Note Overview. Each pot represents one individual irrigation zone capable of watering a crop field](assets/Thumbnail.png)
+
+Implementing traditional wired communication infrastructure in remote areas can be expensive and time-consuming. LoRaWAN®, being a wireless technology, provides a cost-effective alternative as it requires minimal infrastructure setup, reducing installation and maintenance costs.
+
+Arduino has you covered in these scenarios with its Pro solutions. With products designed to work in remote environments, supplying their power from renewable sources and providing long-distance connectivity and low power consumption.
+
+The shown application note is intended to replicate a scaled smart farming application, that can be implemented on real agriculture fields using the same hardware and firmware.
+
+## Goals
+
+The goal of this application note is to showcase a sensorized farming irrigation system using a combination of an Edge Control, an MKR WAN 1310, and the Arduino IoT Cloud. The project's objectives are the following:
+
+- Independently control four irrigation zones using latching valves.
+- Leverage MKR WAN 1310 with LoRa® and a Wisgate (Lite or PRO) to communicate with Arduino IoT Cloud.
+- Monitor soil moisture and decide whether to irrigate based on it. 
+- Display the soil humidity level on the Edge Control Enclosure kit LCD.
+- Manually activate irrigation through Enclosure Kit built-in push button.
+- Monitor average humidity level, irrigation time and water consumption on dedicated charts on Arduino IoT Cloud.
+- Get water from a garden hose with a flow sensor able to evaluate the amount of consumed water.
+
+## Hardware and Software Requirements
+
+![Project main hardware](assets/HARDWARE.png)
+
+### Hardware Requirements
+- Arduino Edge Control
+- Arduino MKR WAN 1310
+- Arduino Edge Control Enclosure Kit
+- Water flow sensor (YF-B2 DN15)
+- WATERMARK Soil Moisture Sensors.
```

Comment:
Add quantities to these items too please I know it is obvious for the reader the quantities that they need to use but please remember how companies and organization works... this can end in an email to another department that probably don't know anything about this stuff and they are not putting attention.

If our user can literally copy and paste this section into an email we are doing an excellent job. cc @mcmchris @TaddyHC @canchebagur